core/engine/src/value/conversions/nullable.rs

```diff
@@ -0,0 +1,314 @@
+//! A value that can be `null` in JavaScript, but not `undefined`.
+
+use crate::value::{TryFromJs, TryIntoJs};
+use boa_engine::{Context, JsResult, JsValue};
+
+/// A value that can be `null` in JavaScript, but not `undefined`.
+///
+/// This is used to distinguish between values in JavaScript that can be
+/// `null` (i.e. `Nullable<T>`) vs. those of which that can be `undefined`
```

Comment:
```suggestion
/// `null` (i.e. [`Nullable<T>`]) vs. those of which that can be `undefined`
```

Nitpick (non-blocking): And all places where it applies. Has the advantage of creating a link in docs in the event we rename something, rust-analyzer renames them too.